server/route/api/v2/memo_service.go

```diff
@@ -450,6 +450,11 @@ func (s *APIV2Service) GetUserMemosStats(ctx context.Context, request *apiv2pb.G
 	}
 	stats := make(map[string]int32)
 	for _, memo := range memos {
```

Comment:
Normally, `s.Store.ListMemos(ctx, memoFind)` should not return a slice that includes nil.